run_benchmarks.sh

```diff
@@ -0,0 +1,28 @@
+TMPDIR="${PWD}/bundles"
+if [ -d "$TMPDIR" ]; then
+  # Take action if $DIR exists. #
+  echo "removing ${TMPDIR} before running benchmarks"
+  rm -rf ${TMPDIR}
+fi
+mkdir ${TMPDIR}
+
+DIR="rpm-adapter"
+if [ -d "$DIR" ]; then
+  # Take action if $DIR exists. #
+  echo "remove ${DIR} before running benchmarks"
+  exit 0
+fi
+echo "Currently TMP: ${TMPDIR}"
+#cp /mnt/disk2/projects/rpmtests/bundle1000.tar.gz ${TMPDIR}
+#cp /mnt/disk2/projects/rpmtests/bundle100.tar.gz ${TMPDIR}
+
+git clone https://github.com/artipie/rpm-adapter.git
```

Comment:
@milkysoft this script is a part of repository, so I think we can assume the repository was already cloned here

Reply:
@g4s8 , fixed with latest [commit](https://github.com/artipie/rpm-adapter/pull/386/commits/7607ca7c554416dcdf9bcb7723af6fc9e1f3c09b)